scripts/install_and_test_openqaoa_dev.sh

```diff
@@ -1,9 +1,6 @@
 #Exit immediately if a command exits with a non-zero status.
 set -e
 
-# TODO: Dynamically generate names (The order of installing is important here.)
-modulesList=("src/openqaoa-core" "src/openqaoa-qiskit" "src/openqaoa-braket" "src/openqaoa-pyquil" "src/openqaoa-azure")
-
 pip install .
-pytest tests/ src/*/tests
-pip uninstall -y openqaoa
\ No newline at end of file
+pytest tests/ src/*/tests -n auto
+pip uninstall -y openqaoa-meta
```

Comment:
I believe the `meta` in the name needs to go

Reply:
Good catch. Removed `meta` reference.